src/components/elements/EventMapView.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { Map, Marker } from "pigeon-maps";
+
+function EventMapView({ events }) {
+
+    // Example list of events
+    events = [
```

Comment:
Can we indent all the code 2 spaces. I don't have the linter set up to enforce style yet, but right now the could is many indented 2 spaces.

Reply:
In general, it is not good practice to reassign a variable passed in as a prop. Can we rename this variable for now. Maybe something like `const mockEvents` for now and use them in the component below. Then I think we'll be good to merge and iterate on this component in a separate pull request. Thanks!